lib/gitlab/client/branches.rb

```diff
@@ -82,7 +88,6 @@ def create_branch(project, branch, ref)
     #
     # @param  [Integer, String] project The ID or name of a project.
     # @param  [String] branch The name of the branch to delete
-    # @return [Gitlab::ObjectifiedHash]
```

Comment:
It seems to always return `false` (Although I'm not sure where that comes from!)... so I figure it must be an incidental return value that is not of any real significance.  (The API documentation doesn't mention any specific return value either)